src/components/graph-fieldset/index.tsx

```diff
@@ -167,13 +167,13 @@ const GraphFieldset: FC<Props> = ({
             labelPosition={LabelPosition.RIGHT}
             onClick={() => handleOnClickInputType(InputType.FILE)}
           >
-            <SC.InputType>
+            <SC.InputType id='validateFile'>
```

Comment:
Vet ikke hvor sannsynlig det er her, men kan risikere id-kollisjon når id-en er hardkodet.
Fra React 18 kan `useId` brukes: https://beta.reactjs.org/reference/react/useId#generating-unique-ids-for-accessibility-attributes , men dette prosjektet bruker React 17.

Ville sett slik ut:
```
const validateFileId = useId()
...
<SC.InputType id=validateFileId >
```

Et alternativ er at du lager et issue på at vi bør bruke `useId`, men som har som avhengighet at vi må oppgradere til React 18 først.

Hva tenker du @jeffreiffers ?

Reply:
Ja, husker at en av wcag (nivå A) feilene handlet nettop om noe slikt. At noen svg-filer (ikon) hadde lik id. Husker at jeg brukte lodash sin uniqueId() metode siden den allerede var installert. Men dette var da i portal som nå er oppgradert til react 18, og nå kan også bruke useId() som du påpeker. 👍